### PR TITLE
Don't pass in intervals file to freec

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ Rapaselet is a delta formed by the Rapaätno river between the Bielloriehppe mas
 - [#1203](https://github.com/nf-core/sarek/pull/1203) - Fix issue with Singularity containers on test profiles
 - [#1204](https://github.com/nf-core/sarek/pull/1204) - Fix issue with nf-validation: lane can be a requirement of bam too now
 - [#1205](https://github.com/nf-core/sarek/pull/1205) - Less tests triggered
+- [#](https://github.com/nf-core/sarek/pull/) - Don't pass in intervals file to ControlFREEC for WGS analysis
```
Comment (FriederikeHanssen) [inline CHANGELOG.md:61]:
```suggestion
- [#1214](https://github.com/nf-core/sarek/pull/1214) - Don't pass in intervals file to ControlFREEC for WGS analysis
```

**Review by maxulysse — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (maxulysse) [reply CHANGELOG.md]: That's lazy

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting

**Comment by asp8200** [comment]
LGTM - just waiting for tests to complete.

**Pull request merged**